samples/snippets/noxfile.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2019 Google LLC
```

Comment:
This file is templated: https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_samples/noxfile.py.j2